ui/package.json

```diff
@@ -32,6 +32,7 @@
     "react-icons": "^4.6.0",
     "react-monaco-editor": "^0.50.1",
     "react-moveable": "^0.40.0",
+    "react-resizable": "^3.0.4",
```

Comment:
Need this:

```
yarn add -D @types/react-resizable
```

Otherwise the compiler complains:

```
ERROR in src/components/Canvas.tsx:37:30
TS2307: Cannot find module 'react-resizable' or its corresponding type declarations.
    35 |
    36 | import Moveable from "react-moveable";
  > 37 | import { ResizableBox } from "react-resizable";
       |                              ^^^^^^^^^^^^^^^^^
    38 | import Ansi from "ansi-to-react";
    39 |
    40 | import { customAlphabet } from "nanoid";
```

Reply:
I was wondering why...I shelled into the UI instance to manually install react-resizable to fix.